### PR TITLE
Reports + solución de schedule

### LINKED ISSUE #144: Vista de reportes
Actualizar la vista de reportes de acuerdo al nuevo modelo. 

Ahora sacamos el link de la barra para que no se pueda acceder y cuando esté listo lo volvemos a poner

### LINKED ISSUE #177: Schedule start_time and end_time

### PR DESCRIPTION
**Pull request by FedeG**
Close #144
Close #177

### CODE REVIEW

File: manager/templates/reports/dashboard.html [hunk 1]
```diff
@@ -24,9 +24,13 @@
...
             <div class="panel-body">
-                <i class="fa fa-users" style="color: #0b62a4"> {% trans "Attendees Confirmed" %} </i><br>
-                <i class="fa fa-users" style="color: #FA5858"> {% trans "Attendees Not Confirmed" %}</i><br><br>
-                <div id="attendee"></div>
+                {% if has_attendee or has_installation_attendee%}
+                    <i class="fa fa-users" style="color: #0b62a4"> {% trans "Attendees Confirmed" %} </i><br>
+                    <i class="fa fa-users" style="color: #FA5858"> {% trans "Attendees Not Confirmed" %}</i><br><br>
+                    <div id="attendee"></div>
+                {% else %}
+                    <strong>{% trans "Hasn't registered attendees" %}</strong>
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There aren't any registered attendees

File: manager/templates/reports/dashboard.html [hunk 2]
```diff
@@ -37,10 +41,15 @@
...
-                <i class="fa fa-users" style="color: #4da74d"> {% trans "Speakers" %}</i>
-                <div id="staff"></div>
+                {% if has_organizer or has_installers or has_speakers or has_collaborators %}
+                    <i class="fa fa-users" style="color: #FFBB13"> {% trans "Organizers" %}</i><br>
+                    <i class="fa fa-users" style="color: #0b62a4"> {% trans "Collaborators" %}</i><br>
+                    <i class="fa fa-users" style="color: #FA5858"> {% trans "Installers" %}</i><br>
+                    <i class="fa fa-users" style="color: #4da74d"> {% trans "Speakers" %}</i>
+                    <div id="staff"></div>
+                {% else %}
+                    <strong>{% trans "Hasn't registered staff" %}</strong>
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There isn't any registered staff

File: manager/templates/reports/dashboard.html [hunk 3]
```diff
@@ -51,7 +60,11 @@
                 <strong>{% trans "Counts" %}</strong>
             </div>
             <div class="panel-body">
-                <div id="counts"></div>
+                {% if has_talk_proposals or has_installations %}
+                    <div id="counts"></div>
+                {% else %}
+                    <strong>{% trans "Hasn't registered talks or installations" %}</strong>
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There aren't any talk or installations submitted

Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There aren't any talks or installations submitted

File: manager/templates/reports/dashboard.html [hunk 4]
```diff
@@ -62,29 +75,43 @@
                 <strong>{% trans "Installations" %}</strong>
             </div>
             <div class="panel-body">
-                <div id="installations"></div>
+                {% if has_installations %}
+                    <div id="installations"></div>
+                {% else %}
+                    <strong>{% trans "Hasn't registered isntallations" %}</strong>
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There aren't any registered installations

File: manager/templates/reports/dashboard.html [hunk 5]
```diff
@@ -62,29 +75,43 @@
...
         <div class="panel panel-primary">
             <div class="panel-heading text-center">
                 <strong>{% trans "Votes For Talks" %}</strong>
             </div>
             <div class="panel-body">
-                <div id="votes"></div>
+                {% if has_talk_proposals %}
+                    <div id="votes"></div>
+                {% else %}
+                    <strong>{% trans "Hasn't registered talk proposals" %}</strong>
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There aren't any talk proposals submitted

File: manager/templates/reports/dashboard.html [hunk 6]
```diff
@@ -62,29 +75,43 @@
...
         <div class="panel panel-primary">
             <div class="panel-heading text-center">
                 <strong>{% trans "Installer For Level" %}</strong>
             </div>
             <div class="panel-body">
-                <div id="installers"></div>
+                {% if has_installers %}
+                    <div id="installers"></div>
+                {% else %}
+                    <strong>{% trans "Hasn't registered isntallations" %}</strong>
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: There aren't any registered installations

File: manager/templates/reports/dashboard.html [hunk 7]
```diff
@@ -99,46 +126,53 @@
...
+                Q([$.get("/api/attendee/?format=json&event={{ event.pk }}&reduce=True"),
+                  $.get("/api/installationattendees/?format=json&event={{ event.pk }}&reduce=True"),
+                  $.get("/api/nonregisteredattendees/?format=json&event={{ event.pk }}&reduce=True")])
+                  .spread(function (attendees, installation_attendees, non_registered_attendees){
+                    var confirmed = attendees.confirmed + installation_attendees.confirmed + non_registered_attendees.total;
+                    var not_confirmed = attendees.not_confirmed+installation_attendees.not_confirmed;
+                    Morris.Donut({
+                      element: 'attendee',
+                      data: [
+                        {label: "{% trans "Attendees Confirmed" %}", value: confirmed, color: "#0b62a4"},
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: Confirmed Attendees

File: manager/templates/reports/dashboard.html [hunk 8]
```diff
@@ -99,46 +126,53 @@
...
+                        {label: "{% trans "Attendees Not Confirmed" %}", value: not_confirmed, color: "#FA5858"}
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: Unconfirmed Attendees

File: manager/templates/reports/dashboard.html [hunk 9]
```diff
@@ -99,46 +126,53 @@
...
+                    for (var item in event_report.votes_for_talk){
+                        votes.push({y: item, a: event_report.votes_for_talk[item]});
+                    };
+
+                    Morris.Bar({
+                      element: 'votes',
+                      data: votes,
+                      xkey: 'y',
+                      ykeys: ['a'],
+                      labels: ['{% trans "Count" %}']
```
Comment (reyiyo) [inline manager/templates/reports/dashboard.html]: Cambiamos "Count" por "Total" en todos lados? Quedaría bien?

File: manager/views.py [hunk 10]
```diff
@@ -319,7 +323,9 @@ def schedule(request, event_slug):
         return render(request, 'activities/schedule.html',
                       update_event_info(event_slug, event=event, render_dict={'schedule': schedule}))
     messages.warning(
-        _("You don't have confirmed activities, please confirm the activitiees and then confirm the schedule"))
+        request,
+        _("You don't have confirmed activities, please confirm the activitiees and then confirm the schedule")
```
Comment (reyiyo) [inline manager/views.py]: You don't have any confirmed activities. Please confirm the activities first and then confirm the schedule

File: manager/templates/reports/dashboard.html (hunk 9 above)
Comment (FedeG) [inline manager/templates/reports/dashboard.html]: Si, ahora lo cambio

**Pull request merged**